botorch/acquisition/input_constructors.py

```diff
@@ -881,42 +884,38 @@ def construct_inputs_EHVI(
         if alpha is None
         else alpha
     )
-    # This selects the objectives (a subset of the outcomes) and set each
-    # objective threshold to have the proper optimization direction.
-    if objective is None:
-        objective = IdentityAnalyticMultiOutputObjective()
-    if isinstance(objective, RiskMeasureMCObjective):
-        pre_obj = objective.preprocessing_function
-    else:
-        pre_obj = objective
-    ref_point = pre_obj(objective_thresholds)
 
     # Compute posterior mean (for ref point computation ref pareto frontier)
     # if one is not provided among arguments.
     if Y_pmean is None:
         with torch.no_grad():
             Y_pmean = model.posterior(X).mean
+
     if alpha > 0:
         partitioning = NondominatedPartitioning(
-            ref_point=ref_point,
-            Y=pre_obj(Y_pmean),
```

Comment:
It looks like we lost this logic in the diff: namely, applying an objective to the incumbent Y in the case of q(Log)EHVI

Reply:
Good catch! forgot to push a commit :)